### DISCUSSION TITLE
Syntax highlighting color schemes

### DISCUSSION THREAD

**Discussion by steff75** [Waboodoo/HTTP-Shortcuts#359] [Ideas]
Hello,
I have a question regarding the 'Syntax highlighting'. Could you please offer other color schemes?

I'm using the dark theme and I'm having a lot of trouble with the readability of the gray used for parentheses, operators and more.  (My eyes are slowly getting worse and recently I need reading glasses)

![20230617_182609 (Klein)](https://github.com/Waboodoo/HTTP-Shortcuts/assets/19283026/acefb2c9-0f30-4fe6-b13c-6d0ecd2af6ae)

If you could use a scheme with as "screaming colors" as possible, like pink, that would be great. Possibly it would be also already better if the background of the textbox would be a real black.

On this occasion I would also like to see the scripting editor itself with an execute button. Then you could try out while writing in between, whether you are on the right track.

Otherwise, I find your app great and use it for many years now.
Thank you very much!

**Comment by steff75** [comment]
I have seen that my ideas have already been implemented in the new version.

Thank you very much 👍🙂

**Reply by Waboodoo** [reply]
I had picked the color for the original color scheme somewhat arbitrarily, so I figured I would just quickly adjust them a bit for better contrast. Glad to hear it made things better.
